crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -1817,6 +1817,29 @@ impl Capsule2d {
             half_length: length / 2.0,
         }
     }
+
+    /// Get the part connecting the semicircular ends of the capsule as a [`Rectangle`]
+    #[inline(always)]
+    pub fn to_rectangle(&self) -> Rectangle {
+        Rectangle::new(self.radius * 2.0, self.half_length * 2.0)
+    }
+}
+
+impl Measured2d for Capsule2d {
+    /// Get the area of the capsule
+    #[inline(always)]
+    fn area(&self) -> f32 {
+        // pi*r^2 + (2r)*l
+        PI * self.radius.squared() + self.to_rectangle().area()
+    }
+
+    /// Get the perimeter or circumference of the capsule
+    #[inline(always)]
+    #[doc(alias = "circumference")]
```

Comment:
Whoops didn't include updating the doc comment in my previous suggestion, I meant this:
```suggestion
    /// Get the perimeter of the capsule
    #[inline(always)]
```

Reply:
Good point, I copied it from `Circle` without thinking